docs/Developers/Writing_Native_Bears.rst

```diff
@@ -411,6 +411,16 @@ strings as a value:
     class SomeBear(Bear):
         LANGUAGES = {'C', 'CPP','C#', 'D'}
 
+.. important::
```

Comment:
The same content is repeated twice (in linter bear and native bear file). This approach might be acceptable (though I tend to reject) but is not ideal.

Could you try and see if this piece of content could be reused? https://sublime-and-sphinx-guide.readthedocs.io/en/latest/reuse.html might help.

Reply:
@li-boxuan Thank you sir.